typescript/infra/src/config/agent.ts

```diff
@@ -472,16 +446,12 @@ export class ChainAgentConfig<Chain extends ChainName> {
     return this.awsKeys;
   }
 
-  get kathyConfig(): KathyConfig | undefined {
-    if (!this.agentConfig.kathy) {
-      return undefined;
-    }
-    return getChainOverriddenConfig(this.agentConfig.kathy, this.chainName);
-  }
-
   get kathyEnabled() {
```

Comment:
likewise I think you can safely remove the rest of the kathy stuff in this class